pkg/gui.py

```
@@ -64,20 +82,19 @@ def on_close(self):
         self.lock_frame(False)
-        # self.show()
 
-    def hide(self):
+    def hide(self) -> None:
         '''
         hides main frame
         '''
         self.root.withdraw()
 
-    def lock_frame(self, enable):
+    def lock_frame(self, enable) -> None:
```

Ruff (0.8.0)

Missing type annotation for function argument `enable`

(ANN001)